### PR TITLE
Added sidebar menu on Privacy page

### PR DESCRIPTION
**Pull request by shreyanshdwivedi**
**Changes proposed in this pull request**
- Added sidebar on `Privacy` page

**Screenshots (if appropriate)** 
screenshot_2018-12-20 loklak search - distributed search for twitter and social media with peer to peer technology

**Link to live demo: **  http://pr-942-fossasia-loklaksearch.surge.sh

**Related to #937**

### CODE REVIEW

**Comment by shreyanshdwivedi** [comment]
@praveenojha33 @simsausaurabh please review :)

File: src/app/privacy/privacy.component.html [hunk 1]
```diff
@@ -1,70 +1,173 @@
...
+					any changes. We will notify you of any changes by posting the new Privacy Policy on this page. These changes are
+					effective immediately after they are posted on this page.
+				</p>
+			</div>
+			<h2>Contact Us</h2>
+			<p>For information about how to contact loklak, please visit our <a routerLink="/contact">contact page</a>.
+			</p>
+		</div>
+	</div>
+	<app-footer></app-footer>
```
Comment (simsausaurabh) [inline src/app/privacy/privacy.component.html]: Add EOF.

**Review by simsausaurabh — COMMENTED** [review]
Add EOF and rest is good.

**Comment by shreyanshdwivedi** [comment]
@simsausaurabh @praveenojha33 please review :)

**Review by praveenojha33 — APPROVED** [review]
LGTM :+1:

**Review by simsausaurabh — CHANGES_REQUESTED** [review]
Margin between main screen content from side bar on Privacy page is smaller than compared to your other PR which adds sidebar on Terms page. Please change the current one to be similar to the one on Terms page.

**Comment by shreyanshdwivedi** [comment]
@simsausaurabh I've made the required changes and squashed the commits too. Please review :)

**Comment by simsausaurabh** [comment]
Resolve conflicts.

**Comment by shreyanshdwivedi** [comment]
@simsausaurabh I've resolved the conflicts. Please merge :))

**Comment by simsausaurabh** [comment]
Squash your commits too, then it will be good to go.

**Comment by shreyanshdwivedi** [comment]
@simsausaurabh I've squashed the commits. Please check :)

**Review by simsausaurabh — APPROVED** [review]

**Pull request merged**